routers/events.router.js

```diff
@@ -135,9 +261,26 @@ router.patch(`/:eventId`, validateId, async (req, res, next) => {
 });
 
 // delete event by id
-router.delete(`/:eventId`, validateId, async (req, res, next) => {
+router.delete(`/:eventId`, validateIds, async (req, res, next) => {
   try {
+    const { user } = req,
+      { eventId } = req.params;
 
```

Comment:
why no const before { eventId} ?

Reply:
We can declare multiple variables at one when we use a comma. Like so:

const var1 = `something`, var2 = `something-else`;